web/src/index.html

```diff
@@ -367,107 +364,120 @@ <h5>Commands</h5>
+            <h2 class="modal-title">Gateway Servers</h2>
+          </div>
+          <div class="modal-body">
+
+            <div class="row">
+              <div class="col col-sm-12">
+                <form id="gateway-server-form">
+                  <table class="table">
+                    <thead>
+                      <tr>
+                        <th>Device ID</th>
+                        <th>UDP Port</th>
+                        <th>Protocol Version</th>
+                        <th>
+                          <button class="btn btn-success gateway-add-btn" id="add-server-btn">
+                            <i class="glyphicon glyphicon-plus"></i>
+                          </button>
+                        </th>
+                      </tr>
+                    </thead>
+                    <tbody id="gateway-server-configs">
+                    </tbody>
+                  </table>
+                  <input type="submit" class="btn btn-success" value="Save" />
+                </form>
+              </div>
+            </div>
+          </div>
+        </div>
       </div>
     </div>
 
     <div>&nbsp;</div>
 
-    <div class="row header-row">
-      <div class="col-sm-12">
-        <h1>Settings</h1>
-      </div>
-    </div>
+    <div id="settings-modal" class="modal fade" role="dialog">
+      <div class="modal-dialog">
+        <!-- Modal content-->
+        <div class="modal-content">
+          <div class="modal-header">
+            <button type="button" class="close" data-dismiss="modal">&times;</button>
+            <h2 class="modal-title">Settings</h2>
+          </div>
+          <div class="modal-body">
 
-    <div>&nbsp;</div>
+            <div>&nbsp;</div>
 
-    <div class="row">
-      <div class="col-sm-12">
-        <form action="#" id="settings">
-          <input type="submit" class="btn btn-success" value="Submit" />
-        </form>
-      </div>
-    </div>
-
-    <div class="row header-row">
-      <div class="col-sm-12">
-        <h1>Sniff Traffic</h1>
+            <div class="row">
+              <div class="col-sm-12">
+                <form action="#" id="settings">
+                  <input type="submit" class="btn btn-success" value="Submit" />
+                </form>
+              </div>
+            </div>
+          </div>
+        </div>
       </div>
     </div>
 
-    <div>&nbsp;</div>
-
-    <div class="row">
-      <div class="col-sm-12">
-        <button type="button" id="sniff" class="btn btn-primary">Start Sniffing</button>
-
-        <div> &nbsp; </div>
-
-        <div id="sniffed-traffic"></div>
-      </div>
+    <div class="alert alert-warning alert-block" id="traffic-sniff">
+      <button type="button" class="close" id="traffic-sniff-close">&times;</button>
+      <h4>Traffic sniffed (sent and received)</h4>
+      <div id="sniffed-traffic"></div>
     </div>
+  </div>
 
-    <div class="row header-row">
-      <div class="col-sm-12">
-        <h1>Admin</h1>
+  <footer>
```

Comment:
I think my preference would be to have at least some of these in the nav bar, styled as links rather than buttons.

Perhaps some of the admin type actions should be buttons in a separate modal.